src/main/kotlin/org/elm/ide/intentions/AddImportIntention.kt

```diff
@@ -268,3 +206,65 @@ data class Candidate(
                     null
     }
 }
+
+object ImportAdder {
```

Comment:
I moved this code unchanged into these static functions. Let me know if I should move it to a separate file, or structure it some other way.